testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeChangeDataFeedCompatibility.java

```diff
@@ -21,6 +21,7 @@
 import io.trino.tempto.BeforeMethodWithContext;
 import io.trino.tempto.assertions.QueryAssert.Row;
 import io.trino.testng.services.Flaky;
+import org.testng.annotations.DataProvider;
```

Comment:
The connector supports dropping a column in case of id or name column mapping mode. Can we add a test dropping a column with CDF?